lizard_connector/connector.py

```diff
@@ -470,6 +475,30 @@ def __init__(self, base="https://demo.lizard.net", username=None,
                 each endpoint parse call.
         """
         self.api_version = version
+        # API Deprecation warning
+        if self.api_version == 2:
+            depracationWarning ="""
+\033[91m============================================================\033[0m
+\033[91m============================================================\033[0m
```

Comment:
I wonder how this looks in a notebook?

I am used to using `warnings.warn(msg)`, and not using any formatting in the msg.

The problem with print statements is that they can't really be handled by outside libraries. With warnings, other people have some control over them.